docs/modules/layers/README.md

```diff
@@ -1,5 +1,8 @@
 # Overview
 
+![deck.gl](https://img.shields.io/badge/deck.gl-v9-green.svg?style=flat-square")
+![WebGPU](https://img.shields.io/badge/webgpu-no-red.svg?style=flat-square")
```

Comment:
```suggestion
![WebGPU not supported](https://img.shields.io/badge/webgpu-no-red.svg?style=flat-square")
```

Reply:
Updated. I think those are just the texts shown when hovered.

Reply:
Yes, but also changes the meaning considerably for users of screen readers. :)